Resources/views/Macros/responsiveImg.html.twig

```diff
@@ -102,9 +102,7 @@
                     data-src="{{ thumbor(image_url, options.placeholder_filter) }}"
                 {% endif %}
                 class="{% if options.lazyload %}lazyload {% endif %}{{ options.class }}"
-                {% if options.alt %}
-                    alt="{{ options.alt }}"
-                {% endif %}
+                alt="{{ options.alt }}"
```

Comment:
Klappt vermutlich, weil als default mit Leerstring belegt, gell?

Reply:
Ja, das geht dann in Ordnung. Mergen, taggen, fertig. Vielleicht noch bei Bonn Process ein `composer update webfactory/responsive-image-bundle`, wenn du extra fleissig sein willst. 😉

Reply:
Ja, absolut!